src/main/java/uk/gov/companieshouse/extensions/api/Application.java

```diff
@@ -11,4 +14,10 @@
   public static void main(String[] args) {
     SpringApplication.run(Application.class, args);
   }
+
+    @PostConstruct
+    public void init(){
+        // Setting Spring Boot SetTimeZone
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
I'm just interested in what the current time-zone is in Spring Boot, before this is set to UTC. Also, whether a unit or integration test could be added to prove that this is working as expected - both in summer and winter time.

Reply:
Added an integration test suite which shows that the change is working. As it tests an actual mongo database the tests are set to @Ignore but can be used to prove the change works if needed.